src/components/ColorGenerator.astro

```diff
@@ -0,0 +1,599 @@
+---
+import { Tabs, TabItem, Code } from "@astrojs/starlight/components";
+
+let rgbCppCode = "rgb_color(128, 128, 128);";
+let rgbCsharpCode = "SplashKit.RGBColor(128, 128, 128);";
+let rgbPythonCode = "rgb_color(128, 128, 128)";
+
+let hsbCppCode = "hsb_color(180.00, 0.50, 0.50);";
+let hsbCsharpCode = "SplashKit.HSBColor(180.00, 0.50, 0.50);";
+let hsbPythonCode = "hsb_color(180.00, 0.50, 0.50)";
+---
+
+<style>
+  /* General slider styles */
+  input[type="range"] {
+    -webkit-appearance: none; /* Remove default appearance */
+    width: 100%; /* Full width */
+    height: 10px; /* Height of the slider */
+    background: #ddd; /* Default background color */
+    border-radius: 5px; /* Rounded edges */
+    outline: none; /* Remove focus outline */
+    transition: background-color 0.3s ease; /* Smooth background color change */
+  }
+
+  /* Individual styling for Red slider */
+  #red-slider {
+    background-color: #f8d7d7; /* Light red background */
+  }
+  #red-slider:hover {
+    background-color: red; /* Intense red on hover */
+  }
+  #red-slider::-webkit-slider-thumb {
+    -webkit-appearance: none; /* Custom thumb styling */
+    width: 25px;
+    height: 25px;
+    background-color: red; /* Thumb color */
+    border-radius: 50%; /* Circular shape */
+    border: 2px solid #000; /* Black border */
+    cursor: pointer; /* Pointer cursor on hover */
+    transition:
+      box-shadow 0.3s ease,
+      transform 0.2s ease; /* Smooth shadow and scaling */
+  }
+  #red-slider:hover::-webkit-slider-thumb {
+    box-shadow: 0 0 10px red; /* Glow effect */
+    transform: scale(1.1); /* Slight enlargement */
+  }
+
+  /* Styling for Green slider */
+  #green-slider {
+    background-color: #d7f8d7; /* Light green */
+  }
+  #green-slider:hover {
+    background-color: green;
+  }
+  #green-slider::-webkit-slider-thumb {
+    background-color: green;
+  }
+  #green-slider:hover::-webkit-slider-thumb {
+    box-shadow: 0 0 10px green;
+    transform: scale(1.1);
+  }
+
+  /* Styling for Blue slider */
+  #blue-slider {
+    background-color: #d7d7f8; /* Light blue */
+  }
+  #blue-slider:hover {
+    background-color: blue;
+  }
+  #blue-slider::-webkit-slider-thumb {
```

Comment:
same here:
`webkit-appearance: none;`

Reply:
Great catch @ctrlvish ! I was able to use this information when finalising this PR :)